plugin/k8s_node/README.md

```diff
@@ -0,0 +1,54 @@
+# k8s_node
+
+## Name
+
+*k8s_node* - resolves node hostname and node IPs from Kubernetes clusters.
+
+## Description
+
+This plugin resolves node external IP and internal IP address(es) of Kubernetes clusters. 
+This plugin is only useful if the *kubernetes* plugin is also loaded.
+
+The plugin resolve node IP addresses. It only handles queries for A and AAAA records;
+all others result in NODATA responses.
+
+## Syntax
+
+~~~
+k8s_node
```

Comment:
Your parse function pulls zones from the plugin declaration.  i.e.  `k8s_node [ZONES...]`